test/unit/mallctl.c

```diff
@@ -1111,7 +1113,7 @@ TEST_BEGIN(test_thread_idle) {
 	 * This alloc and dalloc should leave something in the tcache, in a
 	 * small size's cache bin.
 	 */
-	void *ptr = mallocx(1, 0);
+	void *ptr = mallocx(1, MALLOCX_TCACHE_NONE);
```

Comment:
Why do we not want to use the thread-specific cache?

Reply:
I see. I would say the comments above this line is creating the confusion. Maybe just modify it to add the above explanation in?

Reply:
Just updated. Also added the tcache flush to make everything strictly about the newly created arena.

Reply:
Looks good to me now!